pkg/apis/acme/v1alpha2/types_challenge.go

```diff
@@ -70,8 +70,8 @@ type ChallengeSpec struct {
 	// +optional
 	Wildcard bool `json:"wildcard"`
 
-	// Type is the type of ACME challenge this resource represents, e.g. "dns01"
-	// or "http01".
+	// Type is the type of ACME challenge this resource represents.
+	// One of "http-01" or "dns-01".
 	Type ACMEChallengeType `json:"type"`
```

Comment:
Maybe worth expanding that these 2 are supported by cert-manager but other values exist

Reply:
👍 , but let's keep convo in https://github.com/jetstack/cert-manager/pull/3038#discussion_r450238673